UITests/TransferMethod/AddTransferMethodWireAccountBusinessTests.swift

```diff
@@ -0,0 +1,160 @@
+import XCTest
```

Comment:
We can merge these UI tests for now, but @malaw-hw will continue working on the UI tests. All the wire account UI tests are testing bank account, not wire account.

Reply:
Tests were copy pasted from bank account tests, working on adjusting